### PR TITLE
new update feature for readme file using ChatGPT API

### PR DESCRIPTION
**Pull request by Doreden**
hi its my new feature.
please merge!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
-# git-challenge-match-2024
+# git-challange-match-2024
```
Comment (arikxl) [inline README.md]: typo

**Review by arikxl — CHANGES_REQUESTED** [review]
line #1

File: README.md (hunk 1 above)
Comment (Doreden) [reply README.md]: fixed 🙏

**Review by MikTheGeek — APPROVED** [review]
Look good!

**Review by arikxl — APPROVED** [review]

**Pull request merged**